libs/coin-evm/src/logic.ts

```diff
@@ -253,3 +258,11 @@ export const isNftTransaction = (
   transaction: EvmTransaction,
 ): transaction is EvmTransaction & EvmNftTransaction =>
   ["erc1155", "erc721"].includes(transaction.mode);
+
+/**
+ * Helper adding when necessary a 0
+ * prefix if string lenght is odd
+ */
+export const padHexString = (str: string): string => {
```

Comment:
What is this method usage ? When is the tx value not odd lengthed and opposite ? (same for other usages)

Reply:
A byte is not always represented with 2 numbers, yet that is often expected by blockchains or nano apps